src/MiniCover/Instrumentation/Uninstrumenter.cs

```diff
@@ -9,16 +9,16 @@ public static void Execute(InstrumentationResult result)
         {
             foreach (var assembly in result.Assemblies)
             {
-                if (File.Exists(assembly.BackupFile))
+                if (File.Exists(assembly.Value.BackupFile))
```

Comment:
You can avoid `.Value` by changing the foreach to `foreach (var assembly in result.Assemblies.Values)`